woocommerce/google-listings-and-ads
# Remove Beta Testing UI for production release

### Changes proposed in this Pull Request:

Closes #553 .

This PR removes the beta version of Reclaim URL and Switch URL cards, and use the real release-ready version of the cards. This is done by reverting the changes made in PR https://github.com/woocommerce/google-listings-and-ads/pull/460.

### Screenshots:

Reclaim URL card:

![image](https://user-images.githubusercontent.com/417342/117305385-49785780-aeb1-11eb-8262-797d0324d992.png)

Switch URL card:

![image](https://user-images.githubusercontent.com/417342/117305350-42514980-aeb1-11eb-987a-57226411cb42.png)

### Detailed test instructions:

1. Open https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc
2. Use a Google account and Google MC account that would require reclaiming URL and switching to new URL.
3. Reclaim URL and switch to new URL should not have any mention about beta testing, and users should be able to proceed with reclaim URL and switch URL.

### Changelog Note:

Remove the beta version of Reclaim URL and Switch URL cards, and use the release-ready version of the cards.
